src/recovery.rs

```diff
@@ -779,6 +779,418 @@ mod tests {
         r.collapse_cwnd();
         assert_eq!(r.cwnd(), MINIMUM_WINDOW);
     }
+
+    #[test]
+    fn loss_on_pto() {
+        let mut cfg = crate::Config::new(crate::PROTOCOL_VERSION).unwrap();
+        cfg.set_cc_algorithm(CongestionControlAlgorithm::Reno);
+
+        let mut r = Recovery::new(&cfg);
+
+        let mut now = Instant::now();
+
+        assert_eq!(r.sent[packet::EPOCH_APPLICATION].len(), 0);
+
+        // Start by sending a few packets.
+        let p = Sent {
+            pkt_num: 0,
+            frames: vec![],
+            time_sent: now,
+            size: 1000,
+            ack_eliciting: true,
+            in_flight: true,
+            delivered: 0,
+            delivered_time: std::time::Instant::now(),
+            recent_delivered_packet_sent_time: std::time::Instant::now(),
```

Comment:
```suggestion
            delivered_time: now,
            recent_delivered_packet_sent_time: now,
```

There are many similar lines below as well.

Reply:
Ah yes. Fixed now, thanks.